components/eam/src/physics/cam/physpkg.F90

```diff
@@ -1691,28 +1693,52 @@ subroutine tphysac (ztodt,   cam_in,  &
     nstep = get_nstep()
     call check_tracers_init(state, tracerint)
 
+!!!!!!! OG
+#if 1
```

Comment:
@oksanaguba is this part you want to put in a conditional?

Reply:
yes, this is one of few places with new code.

i will get back to this PR, i need to fix an ERS test with new diagnostics.